tpm2/policy_test.go

```diff
@@ -78,19 +74,20 @@ func TestPcrPolicyCounterHandleSet(t *testing.T) {
 }
 
 func TestCreatePcrPolicyCounterHandle(t *testing.T) {
-	tpm := openTPMForTesting(t)
-	defer closeTPM(t, tpm)
+	tpm, _, closeTPM := tpm2test.OpenTPMConnectionT(t,
+		tpm2test.TPMFeatureOwnerHierarchy|
+			tpm2test.TPMFeatureNV)
+	defer closeTPM()
 
 	testPublic := tpm2.NVPublic{
 		Index:   0x0181fe00,
 		NameAlg: tpm2.HashAlgorithmSHA256,
-		Attrs:   tpm2.NVTypeCounter.WithAttrs(tpm2.AttrNVAuthWrite | tpm2.AttrNVAuthRead),
+		Attrs:   tpm2.NVTypeCounter.WithAttrs(tpm2.AttrNVAuthWrite | tpm2.AttrNVAuthRead | tpm2.AttrNVNoDA),
```

Comment:
the AttrNVNoDA is new here?

Reply:
It is - tests have to request extra features to run commands that could increment the DA counter.